apps/storefront/src/pages/Login/config.ts

```diff
@@ -73,9 +70,9 @@ export const loginCheckout = (data: LoginConfig) => {
   ).then((response) => response.json());
 };
 
-export const sendEmail = (emailAddress: string) => {
```

Comment:
I'm not sold on the renaming of `emailAddress` to `email`, they're different things 🤷 

most of the time the shorthand of `email` is okay-ish, but this one is a bit rough. People would be forgiven for thinking `email: string` in the signature of `sendEmail` was the content of the email, rather than the recipient's address

Reply:
i think I'd call that body or content... but fair point